homeassistant/components/fibaro/diagnostics.py

```diff
@@ -0,0 +1,56 @@
+"""Diagnostics support for fibaro integration."""
+
+from __future__ import annotations
+
+from collections.abc import Mapping
+from typing import Any
+
+from pyfibaro.fibaro_device import DeviceModel
+
+from homeassistant.components.diagnostics import async_redact_data
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceEntry
+
+from . import CONF_IMPORT_PLUGINS, FibaroConfigEntry
+
+TO_REDACT = {"password"}
```

Comment:
where do we see this? I dont see this in the snapshot

Reply:
Yes right, better to cover that in a test. I appended it in the test data so now it is tested as well.